runtime-modules/proposals/codex/src/types.rs

```diff
@@ -45,6 +46,7 @@ pub enum ProposalDetails<
     OpeningId,
     ProposalId,
     UpdateChannelPayoutsParameters,
+    TokenTokenConstraints,
```

Comment:
```suggestion
    TokenConstraints,
```
I don't think the actual type `TokenConstraints` is in scope, so we can reuse it for the template parameter name here.